README.md

```diff
@@ -1,27 +1,26 @@
 # Operator
 
-The RancherOS operator is responsible for managing the RancherOS versions
+The Elemental operator is responsible for managing the Elemental versions
 and maintaining a machine inventory to assist with edge or baremetal installations.
 
 ## Installation
 
-The RancherOS operator can be added to a cluster running Rancher Multi Cluster
+The Elemental operator can be added to a cluster running Rancher Multi Cluster
 Management server.  It is a helm chart and can be installed as follows:
 
 ```bash
-helm -n cattle-rancheros-operator-system install --create-namespace rancheros-operator https://github.com/rancher/os2/releases/download/v0.1.0-alpha12/rancheros-operator-0.1.0-alpha12-amd64.tgz
+helm -n cattle-rancheros-operator-system install --create-namespace rancheros-operator https://github.com/rancher-sandbox/rancheros-operator/releases/download/v0.1.0/rancheros-operator-0.1.0.tgz
```

Comment:
we need a release from this repository. But I think this should came after we set the drone CI cc @paynejacob

Reply:
Drone ci?

Reply:
yeap, for official images that is the way to go to have them published on dockerhub under the rancher org :man_shrugging: